hot-desk/src/main/resources/db.changelog/changes/booking.yaml

```diff
@@ -77,60 +77,4 @@ databaseChangeLog:
           tableName: booking
       changes:
         - dropTable:
-            tableName: booking
-
-  - changeSet:
-      id: create table booking
-      author: asilbekdev@gmail.com
```

Comment:
we can't delete existing changesets

Reply:
We know that the sequence of changelogs in the master yaml is important. It caused the same changeset and a number of errors because it was in the same file. We agreed with the team that we should open another new file